tasks/update-cnb-dependency/main.go

```diff
@@ -49,57 +50,50 @@ func updateCNBDependencies() error {
 		return errors.Wrap(err, "failed to load task resources")
 	}
 
-	deprecationDate, err := NewDependencyDeprecationDate(flags.deprecationDate, flags.deprecationLink, config.Dep.ID, flags.versionLine, flags.deprecationMatch)
-	if err != nil {
-		return errors.Wrap(err, "failed to create deprecation date")
-	}
-
+	// Load, update and save dependencies
```

Comment:
Could we choose different names for these methods/variables to remove the need for this comment?

Reply:
I moved these methods to buildpack_toml with a clearer method name, to remove the comments